src/openfermion/resource_estimates/__init__.py

```diff
@@ -10,10 +9,10 @@
 #   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
-#coverage: ignore
-import pytest
 
 try:
+    import jax
     import pyscf
-except ImportError:
-    pytest.skip('Need pyscf for resource estimates', allow_module_level=True)
+    HAVE_DEPS_FOR_RESOURCE_ESTIMATES = True
+except ModuleNotFoundError:
```

Comment:
what's the difference between `ImportError` and `ModuleNotFoundError`? I usually use import error for this. Is one a subclass of another?

Reply:
I think ModuleNotFoundError is designed to catch a narrower set of errors (can't find the module), if the module is there but there's another error raised when importing it it would raise an ImportError. So I guess ImportError will catch a wider range of issues with the two dependencies.

Reply:
ah good point. Maybe print out a message if you think that could be helpful?